nomad_darwin.go

```diff
@@ -31,12 +31,19 @@ func init() {
 }
 
 func readDefaultForNoMAD(key string) (string, error) {
-	cmd := execCommand("defaults", "read", "com.trusourcelabs.NoMAD", key)
-	out, err := cmd.Output()
-	if err != nil {
-		return "", err
+	// check for key in managed preferences first
+	cmdManaged := execCommand("defaults", "read", "/Library/Managed Preferences/com.trusourcelabs.NoMAD.plist", key)
+	outManaged, errManaged := cmdManaged.Output()
```

Comment:
generally you should always call your error variable `err` unless you actually need two errors to co-exist.

I would also still just use `cmd` and `out` as you can reuse them, or since you're using `:=` on the inner scope, redefine them.